tests/codegen/ccode/scripts/arrays.py

```diff
@@ -0,0 +1,40 @@
+from pyccel.decorators import types
+import pytest
+#==============================================================================
+
+@types( 'int[:,:](order=C)' )
+def double_loop_on_2d_array_C( z ):
+
+    from numpy import shape
+
+    s = shape( z )
+    m = s[0]
+    n = s[1]
```

Comment:
We can now write `m, n = shape(z)`, so I suggest we use that syntax in our tests